package.json

```diff
@@ -2,21 +2,30 @@
   "name": "@orchidjs/unicode-variants",
   "version": "1.0.4",
   "description": "Unicode variant string matching",
-  "main": "dist/umd/index.js",
-  "browser": "dist/umd/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/types/index.d.mts",
   "repository": {
     "type": "git",
     "url": "https://github.com/orchidjs/unicode-variants.git"
   },
+  "type": "module",
+  "main": "dist/cjs/index.js",
```

Comment:
Change: `main` now points to `dist/cjs` instead of `dist/umd`. The UMD build should be reserved for browser use.